pkgs/by-name/mt/mtk_uart/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  libusb1,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "mtk_uartboot";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "981213";
+    repo = "mtk_uartboot";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-mwwm2TVBfOEqvQIP0Vl4Q2SkcZxX1JP7rShmjaY+pWE=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-DtYCSPcyLDYeo9fIQpHGdm5r6ijRAzsDExWcDuSvh/o=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    libusb1
+  ];
+
+  meta = {
+    description = "A tool for MediaTek UART boot mode";
```

```suggestion
    description = "Tool for MediaTek UART boot mode";
```
Not start with the definite or an indefinite article.